docs/core/install/sdk.md

```diff
@@ -48,34 +48,9 @@ export PATH=$PATH:$HOME/dotnet
 
 ::: zone pivot="os-linux"
```

Comment:
Is this pivot still applicable?

Reply:
Not really, but since it was the current experience I want to keep it until I follow up with a PR redesign the entire page as I'm going to break out windows and macos too.